pages.bn/linux/free.md

```diff
@@ -0,0 +1,20 @@
+# free
+
+> সিস্টেমে `Free` এবং ব্যবহৃত মেমোরির মান দেখায়
+> আরো তথ্যের জন্য: <https://manned.org/free>.
```

Comment:
```suggestion
> আরও তথ্য পাবেন: <https://manned.org/free>.
```

Reply:
আরো is more natural than আরও. If you still want the change, I will correct this